### PR TITLE
[EX-51] Create categories api endpoint

### PR DESCRIPTION
**Pull request by Arabadzhiew**
Closes EX-51

### CODE REVIEW

**Review by IvayloSlavchev — APPROVED** [review]
LGTM!

**Review by petya0111 — APPROVED** [review]

File: server/services/CategoryService.js [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+            where: {
+                id: categoryId
+            }
+        }).catch(error => {
+            console.log(error);
+            return new Error(500, error.message);
+        });
+
+        if(category === null){
+            return false;
```
Comment (JordanRad) [inline server/services/CategoryService.js:51]: `return true`  & `return false` leads to HTTP Response true/false, which is a bit confusing to the user. You may use a message like "Category has been deleted successfully"

Comment (petya0111) [reply server/services/CategoryService.js:51]: I think this is not a problem. The user does not interact with the backend directly but with the frontend. I think the front end can add this message for the user. @JordanRad  @Arabadzhiew

Comment (Arabadzhiew) [reply server/services/CategoryService.js:51]: Yeah, as Petya mentioned, the user is not going to interact with our backend api directly. The boolean flag will be used by the frontend to determine the appropriate message that the user will see. Also, I believe this is the way we have implemented our delete methods in all of our other services as well.

**Review by Dukovv — APPROVED** [review]

**Pull request merged**